extensions/Worker.Extensions.CosmosDB/CosmosDBOutputAttribute.cs

```diff
@@ -55,15 +56,9 @@ public CosmosDBOutputAttribute(string databaseName, string collectionName)
         /// <summary>
         /// Optional.
         /// When specified on an output binding and <see cref="CreateIfNotExists"/> is true, defines the throughput of the created
-        /// collection.
-        /// </summary>
-        public int CollectionThroughput { get; set; }
-
-        /// <summary>
-        /// Optional.
-        /// Enable to use with Multi Master accounts.
+        /// container.
         /// </summary>
-        public bool UseMultipleWriteLocations { get; set; }
```

Comment:
This version upgrade will be a breaking change for the customers ?

Reply:
I think it will be: 

https://www.fuget.org/packages/Microsoft.Azure.WebJobs.Extensions.CosmosDB/3.0.9/lib/netstandard2.0/diff/4.0.0-preview2/

Reply:
Yes, this is a new major release.